mambaSharedFramework/Pantos-Generic Playlist Parsing/Pantos-Generic Tag Parsers/GenericSingleValueTagParser.swift

```diff
@@ -20,27 +20,27 @@
 import Foundation
 
 /// Class for generically parsing tags in the form of `#EXT-X-TARGETDURATION:10`, where there is a tag with a one and only one value associated with it
-public class GenericSingleValueTagParser: HLSTagParser {
+public class GenericSingleValueTagParser: PlaylistTagParser {
     
-    let tag: HLSTagDescriptor
-    var singleValueIdentifier: HLSTagValueIdentifier
+    let tag: PlaylistTagDescriptor
+    var singleValueIdentifier: PlaylistTagValueIdentifier
     
     /**
      Constructs a GenericSingleValueTagParser
      
-     - parameter tag: The HLSTagDescriptor for the tag you are trying to parse
-     - parameter singleValueIdentifier: The HLSTagValueIdentifier for the value you are trying to parse out of the tag
+     - parameter tag: The PlaylistTagDescriptor for the tag you are trying to parse
+     - parameter singleValueIdentifier: The TagValueIdentifier for the value you are trying to parse out of the tag
```

Comment:
Did you mean to use `PlaylistTagValueIdentifier` vs. `TagValueIdentifier`?  `PlaylistTagValueIdentifier` is used in other locations (e.g., line 26 above).

Reply:
Yes, fixed. Also fixed the dangling `- parameter` on line 33.